.github/workflows/npm-publish.yml

```diff
@@ -68,6 +81,14 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: Build
-      - run: npm publish --access public
+      - name: bump version in package.json
+        run: |
+          # jq can't replace file contents directly therefore a tmp file is needed
+          tmp_file=$(mktemp)
+
+          cp package.json "$tmp_file" \
+            && jq --arg version "$RELEASE_VERSION" '.version=$version' "$tmp_file" >package.json \
+            && rm -f "$tmp_file"
```

Comment:
jq cannot read and write from the same file, therefore a tmp_file from package.json is needed